roles/win_nodes/kubernetes_patch/tasks/main.yml

```diff
@@ -19,7 +19,7 @@
       register: current_kube_proxy_command
 
     - name: Apply hostnameOverride patch for kube-proxy daemonset
-      shell: "{{bin_dir}}/kubectl patch ds kube-proxy --namespace=kube-system --type=strategic -p \"$(cat hostnameOverride-patch.json)\""
+      shell: "{{bin_dir}}/kubectl patch ds kube-proxy --namespace=kube-system --type=json -p \"$(cat hostnameOverride-patch.json)\""
```

Comment:
I think you can rebase, this has been fixed a few days ago

Reply:
yes.